src/kaleidoscope/plugin/LEDControl.cpp

```diff
@@ -20,8 +20,7 @@
 namespace kaleidoscope {
 namespace plugin {
 
-LEDMode *LEDControl::modes[LED_MAX_MODES];
-uint8_t LEDControl::mode;
+uint8_t LEDControl::mode_id;
```

Comment:
Since it's a private variable, it ought to be `mode_id_`, with a trailing underscore. Might as well make it conform to coding style standards while changing the name.